elements/bulbs-poll/components/request-error.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/prop-types */
```

Comment:
I'm not entirely sure. We may want them here but there are no tests for this component and I didn't want to investigate it now. I'll add a card to track it (https://trello.com/c/BUEeTOmU/275-remove-ignored-lint-rules-in-bulbs-elements).